components/FormWizard/FormWizard.jsx

```diff
@@ -109,7 +106,12 @@ const FormWizard = ({
             saveData(
               formPath,
               updatedData,
-              `${stepId.join('/')}?${getQueryString(queryString)}`
+              title,
+              includesDetails
```

Comment:
We should keep the query string even if the form doesn't have `includesDetails`.

Reply:
We're getting an error if we do